dc-cudami-frontend/cudami-frontend-website/src/main/java/de/digitalcollections/cudami/frontend/website/service/ContentService.java

```diff
@@ -0,0 +1,135 @@
+package de.digitalcollections.cudami.frontend.website.service;
+
+import de.digitalcollections.cudami.client.CudamiClient;
+import de.digitalcollections.cudami.client.exceptions.HttpException;
+import de.digitalcollections.cudami.client.identifiable.entity.CudamiWebsitesClient;
+import de.digitalcollections.cudami.client.identifiable.web.CudamiWebpagesClient;
+import de.digitalcollections.cudami.frontend.website.config.CudamiConfig;
+import de.digitalcollections.model.identifiable.entity.Website;
+import de.digitalcollections.model.identifiable.web.Webpage;
+import de.digitalcollections.model.paging.PageRequest;
+import de.digitalcollections.model.paging.PageResponse;
+import java.lang.invoke.MethodHandles;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Locale;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ContentService {
+
+  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  private final CudamiConfig cudamiConfig;
+  private final CudamiWebpagesClient cudamiWebpagesClient;
+  private final CudamiWebsitesClient cudamiWebsitesClient;
+
+  public ContentService(CudamiConfig cudamiConfig, CudamiClient cudamiClient) {
+    this.cudamiConfig = cudamiConfig;
+    this.cudamiWebpagesClient = cudamiClient.forWebpages();
+    this.cudamiWebsitesClient = cudamiClient.forWebsites();
+  }
+
+  public List<Webpage> getContentPages() {
+    try {
+      // get website
+      Website website = getWebsite();
+
+      List<Webpage> allContentWebpages;
+
+      if (cudamiConfig.getWebpage("content") == null) {
+        // get root webpages
+        PageRequest pageRequest = PageRequest.defaultBuilder().pageSize(100).build();
+        PageResponse<Webpage> rootPagesResponse =
+            cudamiWebsitesClient.getRootPages(website.getUuid(), pageRequest);
+        allContentWebpages = rootPagesResponse.getContent();
+      } else {
+        Pair<Webpage, Locale> webpagePair = getWebpage(cudamiConfig.getWebpage("content"));
+        Webpage webpage = webpagePair.getLeft();
+        allContentWebpages = webpage.getChildren();
+      }
+
+      List<Webpage> activeContentWebpages =
+          allContentWebpages.stream()
+              .filter(
+                  f -> {
```

Comment:
Könnte man noch einen Tick eleganter schreiben:

```suggestion
                  this::isActive
```